src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
You shouldn't use inheritance here. Instead of that it's better to use composition.
So that you will have a class member thas has to be initialized via constructor:
```
private ColorSupplier colorSupplier;
```